lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -40,18 +41,18 @@
 <div class="wrapper-banner wrapper-banner-user">
     <section class="banner banner-user">
         <div class="message message-block message-notice">
-            <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening | h}</h2>
+            <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening}</h2>
             <div class="wrapper-copy-and-actions">
                 <p class="message-copy copy copy-base emphasized">${accomplishment_banner_congrats}</p>
                 <div class="message-actions">
                     <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                     % if facebook_share_enabled:
                       <button class="action action-share-facebook btn-inverse btn-small icon-only" id="action-share-facebook"
                          onclick="FaceBook.share({
-                          share_text: '${facebook_share_text | escapejs}',
+                          share_text: '${facebook_share_text | n, js_escaped_string}',
                           share_link: '${share_url}',
                           picture_link: '${full_course_image_url}',
```

Comment:
We should apply the `js_escaped_string` filter to these URLs as well. I don't believe there should be any problem with javascript-escaping the URLs